official/recommendation/neumf_model.py

```diff
@@ -381,8 +381,8 @@ def compute_eval_loss_and_metrics_helper(logits,              # type: tf.Tensor
   # ignore padded examples
   example_weights *= tf.cast(expanded_metric_weights, tf.float32)
 
-  cross_entropy = tf.losses.sparse_softmax_cross_entropy(
-      logits=softmax_logits, labels=eval_labels, weights=example_weights)
+  cross_entropy = tf.keras.losses.sparse_categorical_crossentropy(
```

Comment:
this change does not seem equivalent. the new one is not using weights. and also i think the original one returns a reduced loss and new one returns per example losses? Does this run?

Reply:
It does, and it seems not to affect the convergence number, weirdly. But the number does fluctuate, so it is hard to tell sometimes. 
Anyway, I changed it to include the example_weights.